include/retdec/common/function.h

```diff
@@ -53,6 +54,7 @@ class Function : public retdec::common::AddressRange
 
 		/// @name Function query methods.
 		/// @{
+		bool isDecompilerDefined() const;
 		bool isUserDefined() const;
```

Comment:
After this PR is merged, could you please prepare a PR for our [IDA plugin](https://github.com/avast/retdec-idaplugin)? It relies on `isUserDefined()` (see [this code](https://github.com/avast/retdec-idaplugin/blob/1b712167bbad1b1d30671a93af74fd3ae5a68d63/src/idaplugin/code_viewer.cpp#L296)).

Reply:
Yes of course, thank you for the notice. Updating plugins is on my list after this. Apart from this IDA plugin should get more updates as @PeterMatula introduced changes to RetDec config.